pynestml/codegeneration/nest_codegenerator.py

```diff
@@ -136,24 +197,24 @@ def generate_module_code(self, neurons: List[ASTNeuron]) -> None:
         if not os.path.exists(FrontendConfiguration.get_target_path()):
```

Comment:
I can't comment directly on line 133/194 (thanks GitHub) but in any case: please break out setting up the namespace to a private method called get_module_namespace().